config

```diff
@@ -109,7 +109,7 @@
     # Pushes the current branch to the remote "origin" (or the remote passed as the parameter) and set it to track the upstream branch
     publish = "!sh -c 'git push -u ${1:-origin} $(git branch-name)' -"
     # Deletes the remote version of the current branch from the remote "origin" (or the remote passed as the parameter)
-    unpublish = "!sh -c 'git push ${1:-origin} :$(git branch-name)' -"
+    unpublish = "!sh -c 'git push ${1:-origin} :$(git branch-name);git branch --unset-upstream $(git branch-name)' -"
```

Comment:
Have you checked that the second command will not execute if the first fails (just a suspicion)? Maybe `&&` would be better?

In general a good idea.